ruby/README.md

```diff
@@ -247,10 +247,44 @@ A developer should be able to view any `it` block in isolation --without its
 DSL methods, it's probably fine.
 
 - Avoid `let`, and `subject` (prefer factory methods)
+- Avoid unnecessary parenthesis in assertions to reduce noise
+
+  ```rb
+  # Good
+  expect(x).to eq y
```

Comment:
I actually prefer the opposite style: I do not like Ruby's inconsistency around parens and prefer to always have parens for method calls (with grumpy exceptions made for some cases like template helpers/macro-y things like `attr_reader`. My personal reasoning is similar to why we always do `,` after the last arg for multi-line formatting: the diff tends to be cleaner when it changes, and you're less likely to accidentally break something just because you forgot to add parens.

But I will defer if there's a general preference for this.